stdlib/tkinter/__init__.pyi

```diff
@@ -352,7 +355,29 @@ class Misc:
     # after_idle is essentially partialmethod(after, "idle")
     def after_idle(self, func: Callable[[Unpack[_Ts]], object], *args: Unpack[_Ts]) -> str: ...
     def after_cancel(self, id: str) -> None: ...
+    if sys.version_info >= (3, 13):
+        def after_info(self, id: str | None = None) -> tuple[str, ...]: ...
+
     def bell(self, displayof: Literal[0] | Misc | None = 0) -> None: ...
+    if sys.version_info >= (3, 13):
+        # Supports options from `_BusyInfo``
+        def tk_busy_cget(self, option: Literal["cursor"]) -> _Cursor: ...
+        busy_cget = tk_busy_cget
+        def tk_busy_configure(self, cnf: Any = None, **kw: Any) -> Any: ...
```

Comment:
You can overload this similarly to other `configure` methods to get rid of `**kw: Any`. Look at `Tk.configure` for a relatively simple example.

After writing the types for those other `configure` methods, I have realized that there are better ways to do it, but that's beyond the scope of this PR and it's fine to go with the existing pattern. By far the most common use case is passing keyword arguments and ignoring the return value, e.g. `widget.busy_config(cursor="hand2")` in this case.